pkg/admission/clusterworkspacetype/admission.go

```diff
@@ -83,5 +84,25 @@ func (o *clusterWorkspaceType) Validate(ctx context.Context, a admission.Attribu
 		return errors.New("organization type can only be created in root workspace")
 	}
 
+	if a.GetOperation() != admission.Update {
+		return nil
+	}
+
+	if a.GetOldObject().GetObjectKind().GroupVersionKind() != tenancyv1alpha1.Kind("ClusterWorkspaceTypes").WithVersion("v1alpha1") {
+		return nil
```

Comment:
🔴  this is dangerous. Don't check for version. Admission should (nearly) always apply to all versions of a GR. Otherwise, this is a time-💣 when we move to v1beta1 or v1.